src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy

```diff
@@ -87,10 +88,23 @@ class SurrealPersistenceService implements PersistenceService {
         final ret3 = surrealDb.sqlAsMap(authorization, "define table wave_scan SCHEMALESS")
         if( ret3.status != "OK")
             throw new IllegalStateException("Unable to define SurrealDB table wave_scan - cause: $ret3")
-        // create wave_scan table
+        // create wave_scan_vuln table
         final ret4 = surrealDb.sqlAsMap(authorization, "define table wave_scan_vuln SCHEMALESS")
         if( ret4.status != "OK")
-            throw new IllegalStateException("Unable to define SurrealDB table wave_scan_vuln - cause: $ret3")
+            throw new IllegalStateException("Unable to define SurrealDB table wave_scan_vuln - cause: $ret4")
+        // create wave_conda_package table
+        try{
+            surrealDb.sqlAsString(authorization, '''
+                                                DEFINE TABLE wave_conda_package SCHEMAFULL; 
+                                                DEFINE FIELD name ON TABLE wave_conda_package TYPE string;
+                                                DEFINE FIELD version ON TABLE wave_conda_package TYPE string;
+                                                DEFINE FIELD channel ON TABLE wave_conda_package TYPE string;
+                                                DEFINE FIELD fullName ON TABLE wave_conda_package TYPE string;
```

Comment:
Not sure `fullName` is really needed. The query could be done using an OR criteria on `name` and `channel` fields

Reply:
I followed this approach 
https://github.com/seqeralabs/wave/pull/403#issuecomment-1947939786

Reply:
if users use 'bioconda::multiqc=1.20' to search for conda packages then fullName will help

Reply:
but if we want to use OR then i can split the serach string and compare different parts with name, channel, and version

Reply:
fair enough